assets/files.wxs

```diff
@@ -1340,9 +1340,6 @@
         <Component Id="cmpB2DC1821DBE648078AEA75AC0AB62408" Guid="{4242bbb5-81dd-481d-a46a-6397d5bce4a5}">
```

Comment:
@SteveL-MSFT, your last commit had 7 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Clipboard cmdlet tests.Unix.-Format Audio: returns error
```powershell
Expected an exception, with FullyQualifiedErrorId 'FailedToGetClipboard,Microsoft.PowerShell.Commands.GetClipboardCommand' to be thrown, but the FullyQualifiedErrorId was 'FailedToGetClipboardUnsupportedFormat,Microsoft.PowerShell.Commands.GetClipboardCommand'. from /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1:133 char:15
    +             { Get-Clipboard -Format $format } | Should -Throw -ErrorI ...
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1: line 133
133:             { Get-Clipboard -Format $format } | Should -Throw -ErrorId 'FailedToGetClipboard,Microsoft.PowerShell.Commands.GetClipboardCommand'
```
Clipboard cmdlet tests.Unix.-Format FileDropList: returns error
```powershell
Expected an exception, with FullyQualifiedErrorId 'FailedToGetClipboard,Microsoft.PowerShell.Commands.GetClipboardCommand' to be thrown, but the FullyQualifiedErrorId was 'FailedToGetClipboardUnsupportedFormat,Microsoft.PowerShell.Commands.GetClipboardCommand'. from /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1:133 char:15
    +             { Get-Clipboard -Format $format } | Should -Throw -ErrorI ...
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1: line 133
133:             { Get-Clipboard -Format $format } | Should -Throw -ErrorId 'FailedToGetClipboard,Microsoft.PowerShell.Commands.GetClipboardCommand'
```
Clipboard cmdlet tests.Unix.-Format Image: returns error
```powershell
Expected an exception, with FullyQualifiedErrorId 'FailedToGetClipboard,Microsoft.PowerShell.Commands.GetClipboardCommand' to be thrown, but the FullyQualifiedErrorId was 'FailedToGetClipboardUnsupportedFormat,Microsoft.PowerShell.Commands.GetClipboardCommand'. from /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1:133 char:15
    +             { Get-Clipboard -Format $format } | Should -Throw -ErrorI ...
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1: line 133
133:             { Get-Clipboard -Format $format } | Should -Throw -ErrorId 'FailedToGetClipboard,Microsoft.PowerShell.Commands.GetClipboardCommand'
```
Clipboard cmdlet tests.Unix.-TextFormatType Text: returns error
```powershell
Expected an exception, with FullyQualifiedErrorId 'FailedToGetClipboard,Microsoft.PowerShell.Commands.GetClipboardCommand' to be thrown, but the FullyQualifiedErrorId was 'FailedToGetClipboardUnsupportedTextFormat,Microsoft.PowerShell.Commands.GetClipboardCommand'. from /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1:119 char:15
    +             { Get-Clipboard -TextFormatType $format } | Should -Throw ...
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1: line 119
119:             { Get-Clipboard -TextFormatType $format } | Should -Throw -ErrorId 'FailedToGetClipboard,Microsoft.PowerShell.Commands.GetClipboardCommand'
```
Clipboard cmdlet tests.Unix.-TextFormatType Rtf: returns error
```powershell
Expected an exception, with FullyQualifiedErrorId 'FailedToGetClipboard,Microsoft.PowerShell.Commands.GetClipboardCommand' to be thrown, but the FullyQualifiedErrorId was 'FailedToGetClipboardUnsupportedTextFormat,Microsoft.PowerShell.Commands.GetClipboardCommand'. from /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1:119 char:15
    +             { Get-Clipboard -TextFormatType $format } | Should -Throw ...
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1: line 119
119:             { Get-Clipboard -TextFormatType $format } | Should -Throw -ErrorId 'FailedToGetClipboard,Microsoft.PowerShell.Commands.GetClipboardCommand'
```

Reply:
Oops, forgot to update the tests.